### PR TITLE
[CARBONDATA-2009][Documentation] Document Refresh command constraint

### PR DESCRIPTION
**Pull request by arshadmohammad**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed? NO
 
 - [ ] Any backward compatibility impacted? NO
 
 - [ ] Document update required? YES

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       NA( This PR has only document change )
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.  NA

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -271,6 +271,7 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   * Before executing this command the old table schema and data should be copied into the new database location.
   * If the table is aggregate table, then all the aggregate tables should be copied to the new database location.
   * For old store, the time zone of the source and destination cluster should be same.
+  * If Carbon table is stored in carbon HIVE meta store then refresh will not register the Carbon table to HIVE meta store catalogue.
```
Comment (sgururajshetty) [inline docs/data-management-on-carbondata.md]: Change all the instances of Carbon to CarbonData

Comment (arshadmohammad) [reply docs/data-management-on-carbondata.md]: Changed

**Comment by sgururajshetty** [comment]
LGTM @chenliang613 kindly review

File: docs/data-management-on-carbondata.md [hunk 2]
```diff
@@ -271,6 +271,7 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   * Before executing this command the old table schema and data should be copied into the new database location.
   * If the table is aggregate table, then all the aggregate tables should be copied to the new database location.
   * For old store, the time zone of the source and destination cluster should be same.
+  * If CarbonData table is stored in CarbonData HIVE meta store then refresh will not register the CarbonData table to HIVE meta store catalogue.
```
Comment (KanakaKumar) [inline docs/data-management-on-carbondata.md]: Please change the point like below to convey the details of reason.

"If old cluster uses hive meta store refresh will not work as schema file is present in file system"

Comment (mohammadshahidkhan) [reply docs/data-management-on-carbondata.md]: @KanakaKumar if schema file is present in the file system then why refresh will not work.
Refresh will not work schema not present in the filesystem.

Comment (KanakaKumar) [reply docs/data-management-on-carbondata.md]: Sorry I mean "If old cluster uses hive meta store, refresh will not work as schema file not exists in file system"

Comment (arshadmohammad) [reply docs/data-management-on-carbondata.md]: Thanks @KanakaKumar for your reviews. I handled the comment. Please have a look

**Comment by KanakaKumar** [comment]
LGTM

**Pull request closed without merging**